guides/common/attributes-foreman-el.adoc

```diff
@@ -1,5 +1,5 @@
 // Overrides for foreman-el build
-:install-on-os: CentOS/RHEL
+:install-on-os: {EL}
```

Comment:
This is now the same in all files, except satellite and foreman-deb. Perhaps declare this in the base attributes file and only keep where it's different?